requirements.in

```diff
@@ -20,7 +20,6 @@ pycryptodome==3.19.1
 pyzmq==22.3.0
 tenacity==8.0.1
 # this package isn't in pypi
```

Comment:
Line 22 (`# this package isn't in pypi`) must also be removed

Reply:
removed